heron/stmgr/src/cpp/manager/stmgr-server.h

```diff
@@ -153,6 +158,9 @@ class StMgrServer : public Server {
   typedef std::map<sp_string, heron::common::AssignableMetric*> QueueMetricMap;
   QueueMetricMap queue_metric_map_;
 
+  typedef std::map<sp_string, heron::common::MultiAssignableMetric*> TuplesMetricMap;
```

Comment:
could you add comment to this field - what this field is for and what key/value in the map means